### ISSUE TITLE
return dom['elements'][0]['authorizationId'] KeyError: 'authorizationId'

### ISSUE THREAD

**Issue by LastSamuray** [coursera-dl/coursera-dl#753]
🚨Please review the [Troubleshooting](../#troubleshooting) section
before reporting any issue. Don't forget to check also the current issues to
avoid duplicates.

### Subject of the issue
I tried to run the script on mac os as usual. After downloading 2 courses I started to get this error. I did not do anything different than the other attempts. But i failed for some reason. Could any of you please help me?

### Your environment
* Operating System (name/version): Mac OS Mojave - 10.14.5
* Python version: 3.8.2
* coursera-dl version: latest one

* Is the problem happening with the latest version of the script?  yes
* Do you have all the recommended versions of the modules? See them in the
  file `requirements.txt`.  yes

### Expected behaviour
Download the course.

### Actual behaviour
Giving me this error, "return dom['elements'][0]['authorizationId']
KeyError: 'authorizationId'"

**Comment by samiullah321** [comment]
![image](https://user-images.githubusercontent.com/39356160/87703053-a9333400-c7b3-11ea-9912-f5a3f3fc2af3.png)
same problem i think its related to notebook platfom

**Comment by samiullah321** [comment]
yep this is caused by notebook download as soon as i remove --download-notebook from argument it works perfectly
